### PR TITLE
Scaling for all pages in the app

### PR DESCRIPTION
**Pull request by salmanmkc**
Adjusted scaling for the whole app and small grammar fix.

## Description
App has values that didn't scale well, for example, if the width was 50, 50*1.25 = 62.5, and it would round up, and cause an offset of 0.5 pixels. I have gone through every page and tried to update the values to scale well.

## Motivation and Context
Make the app scale well.

## How Has This Been Tested?
Building the application

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by chingucoding** [comment]
I think it would be better to merge and #313 and #314 into this PR since they all aim to achieve the same.

**Comment by salmanmkc** [comment]
> I think it would be better to merge and #313 and #314 into this PR since they all aim to achieve the same.

#313 has some changes from #312, but I could merge #314 into this

**Comment by chingucoding** [comment]
> > I think it would be better to merge and #313 and #314 into this PR since they all aim to achieve the same.
> 
> #313 has some changes from #312, but I could merge #314 into this

What I mean is that is better to have one PR that contains all changes from #312,#313 and #314 since they all are fixing the same thing. 😅

File: XamlControlsGallery/ControlPages/TabViewPage.xaml [hunk 1]
```diff
@@ -148,10 +148,10 @@
 
                     <muxc:TabView SelectedIndex="0" Margin="-12" AddTabButtonClick="TabView_AddButtonClick" TabCloseRequested="TabView_TabCloseRequested" Loaded="TabView_Loaded" MinHeight="475" TabWidthMode="SizeToContent">
                         <muxc:TabView.TabStripHeader>
-                            <TextBlock Text="TabStripHeader Content" VerticalAlignment="Center" Margin="8,6" Style="{ThemeResource BaseTextBlockStyle}" />
+                            <TextBlock Text="TabStripHeader Content" VerticalAlignment="Center" Margin="8,8" Style="{ThemeResource BaseTextBlockStyle}" />
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/TabViewPage.xaml:151]: Will this vertical margin be too much? Maybe 4px would be better?

File: XamlControlsGallery/ControlPages/TabViewPage.xaml [hunk 2]
```diff
@@ -148,10 +148,10 @@
...
                         </muxc:TabView.TabStripHeader>
                         <muxc:TabView.TabStripFooter>
-                            <TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="6" Style="{ThemeResource BaseTextBlockStyle}" />
+                            <TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="8" Style="{ThemeResource BaseTextBlockStyle}" />
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/TabViewPage.xaml:154]: Similarly, I worry about messing with the alignment inside the tab strip too much. @stmoy, any toughts?

File: XamlControlsGallery/ItemTemplates.xaml [hunk 3]
```diff
@@ -69,7 +69,7 @@
                     Visibility="{x:Bind BadgeString, Converter={StaticResource nullToVisibilityConverter}}" >
 
                     <Rectangle Fill="{ThemeResource SystemAccentColor}" Opacity=".5" />
-                    <TextBlock Text="{x:Bind BadgeString}" Margin="8,2,8,4" />
+                    <TextBlock Text="{x:Bind BadgeString}" Margin="8,0,8,4" />
```
Comment (YuliKl) [inline XamlControlsGallery/ItemTemplates.xaml:72]: Going from any margin to a 0 margin can be a big jump. Does this impact the text legibility at all?

File: XamlControlsGallery/PageHeader.xaml [hunk 4]
```diff
@@ -80,7 +80,7 @@
                         Click="OnThemeButtonClick"
                         Label="Toggle Theme">
                         <AppBarButton.Icon>
-                            <FontIcon Glyph="&#xE771;" Margin="0,-2,0,0" />
+                            <FontIcon Glyph="&#xE771;" Margin="0,-4,0,0" />
```
Comment (YuliKl) [inline XamlControlsGallery/PageHeader.xaml:83]: I'm always suspicious of negative margins and worry about touching this one. Is this change necessary?

**Comment by stmoy** [comment]
Closing as there are unaddressed comments. @salmanmkc - please feel free to submit a new PR if desired.
